src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -80,7 +80,10 @@ export class CellModel extends Disposable implements ICellModel {
 	private _isParameter: boolean;
 	private _onParameterStateChanged = new Emitter<boolean>();
 	private _isInjectedParameter: boolean;
+	private _previousChartState: any[] = [];
+	private _outputCounter = 0;
```

Comment:
Some comments for these properties would be nice...hard to tell how the `_outputCounter` fits in to everything just by the name...